frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCard.tsx

```diff
@@ -87,9 +97,8 @@ const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => ({
     [theme.breakpoints.down(400)]: {
         padding: theme.spacing(1, 2),
     },
-    '&.Mui-focusVisible': {
-        backgroundColor: theme.palette.background.paper,
-        padding: theme.spacing(0.5, 2, 0.3, 2),
+    '&:focus-visible': {
+        background: theme.palette.table.headerHover,
```

Comment:
Fixes an a11y issue by giving the focused summary a different background color so you know where your tab focus is.